.github/workflows/deploy-testnet.yml

```diff
@@ -115,19 +120,22 @@ jobs:
           for i in "${region_array[@]}"; do
             # Deploy new image version for archivist
             cd ${aleph_path}/aleph-apps/aleph-node-archivists/overlays/testnet/${i}
-            kustomize edit set image "aleph-node-archivist-image-placeholder=${{ env.TESTNET_IMAGE }}"
+            kustomize edit set image \
+              "aleph-node-archivist-image-placeholder=${{ env.TESTNET_IMAGE }}"
 
             # Deploy new image version for validator
             cd ${aleph_path}/aleph-apps/aleph-node-validators/overlays/testnet/${i}
-            kustomize edit set image "aleph-node-validator-image-placeholder=${{ env.TESTNET_IMAGE }}"
+            kustomize edit set image \
+              "aleph-node-validator-image-placeholder=${{ env.TESTNET_IMAGE }}"
           done
 
       - name: GIT | Commit changes to aleph-apps repository.
         uses: EndBug/add-and-commit@v5.1.0
         with:
           author_name: AlephZero Automation
           author_email: alephzero@10clouds.com
-          message: "Updating Testnet docker image tag for pre-release: ${{ steps.vars.outputs.branch }}"
+          message:
```

Comment:
does that work without any extra character at the end of `message:` - meaning the value is in the next line?